payments/tests/conftest.py

```diff
@@ -37,6 +37,35 @@ def two_hour_reservation(resource_in_unit, user):
     )
 
 
+@pytest.fixture
+def requested_reservation_with_order(resource_in_unit, user):
+    reservation = Reservation.objects.create(
+        resource=resource_in_unit,
+        begin=datetime.datetime(2119, 5, 5, 10, 0, 0, tzinfo=UTC),
```

Comment:
Is there a reason our reservation is going to be in the 22nd century? (maybe clearer to just get current date and add timedelta to add a few days, but I guess setting a far future date is more efficient).

Reply:
Changed the times